qiskit/transpiler/target.py

```diff
@@ -655,37 +655,52 @@ def _get_instruction_schedule_map(self):
     def has_calibration(
         self,
         operation_name: str,
-        qargs: tuple[int, ...],
+        qargs: tuple[int, ...] | list[int, ...],
+        operation_params: list[float] | float | None = None,
     ) -> bool:
-        """Return whether the instruction (operation + qubits) defines a calibration.
+        """Return whether the instruction (operation + operation_params + qubits) defines
+        a calibration.
 
         Args:
             operation_name: The name of the operation for the instruction.
-            qargs: The tuple of qubit indices for the instruction.
+            qargs: The qubit indices for the instruction.
+            operation_params: The parameters for the Instruction.
```

Comment:
Please document that in case there are multiple params for a given gate (e.g. `RGate`) the order of the the list in `operation_params` should match the order of the gate params